packages/@aws-cdk/aws-pipes-alpha/lib/logs.ts

```diff
@@ -36,6 +39,64 @@ export enum LogLevel {
   TRACE = 'TRACE',
 }
 
+/**
+ * Log format for `S3LogDestination` logging configuration.
+ *
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-s3logdestination.html#cfn-pipes-pipe-s3logdestination-outputformat
+ */
+export enum S3OutputFormat {
+  /**
+  * Plain text
+  */
+  PLAIN = 'plain',
+  /**
+  * JSON
+  */
+  JSON = 'json',
+  /**
+  * W3C extended log file format
+  * @see https://www.w3.org/TR/WD-logfile
+  */
+  W3C = 'w3c',
+}
+
+/**
+ * Properties for `S3LogDestination`.
+ */
+export interface S3LogDestinationProps {
+  /**
+   * The S3 bucket to deliver the log records for the pipe.
+   *
+   * The bucket can be in the same or a different AWS Account. If the bucket is in
+   * a different acccount, specify `bucketOwner`. You must also allow access to the
+   * Pipes role in the bucket policy of the cross-account bucket.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-s3logdestination.html#cfn-pipes-pipe-s3logdestination-bucketname
+   */
+  readonly bucket: IBucket;
```

Comment:
The documentation you linked here shows that the bucket property isn't required, but we're mandating it here. I'm curious as to why that may be the case.

Reply:
It's not required to create a pipe but it is required if you want to create an `S3LogDestination`. IMO the docs should be updated.